packages/viewer/src/lib/model-views/controller.ts

```diff
@@ -0,0 +1,64 @@
+import { Pager } from '@vertexvis/scene-view-protos/core/protos/paging_pb';
+import { Uuid2l } from '@vertexvis/scene-view-protos/core/protos/uuid_pb';
+import {
+  ListItemModelViewsRequest,
+  ListItemModelViewsResponse,
+} from '@vertexvis/scene-view-protos/sceneview/protos/scene_view_api_pb';
+import { SceneViewAPIClient } from '@vertexvis/scene-view-protos/sceneview/protos/scene_view_api_pb_service';
+import { UUID } from '@vertexvis/utils';
+
+import { createMetadata, JwtProvider, requestUnary } from '../grpc';
+
+export interface ListByItemOptions {
+  cursor?: string;
+  size?: number;
+}
+
+/**
+ * The controller for managing the annotations of a scene and scene view.
+ */
+export class ModelViewController {
+  public constructor(
+    private client: SceneViewAPIClient,
+    private jwtProvider: JwtProvider,
+    private deviceIdProvider: () => string | undefined
+  ) {}
+
+  /**
+   * Fetches a page of model views for a given item. The result includes a
+   * cursor that can be used to fetch the next page of results.
+   *
+   * @param itemId The ID of the scene item to retrieve model views for.
+   * @param options Options to configure fetching of items.
+   * @returns
```

Comment:
nit: looks like this is missing a description of what it returns, could we add one or remove this?